react/components/ProductSummaryBuyButton/ProductSummaryBuyButton.js

```diff
@@ -14,6 +16,18 @@ import displayButtonTypes, {
 
 import productSummary from '../../productSummary.css'
 
+function checkSKUAvailability(seller) {
+  return path(['commertialOffer', 'AvailableQuantity'], seller) > 0
+}
+
+function checkAvailableSKUsInSellers(item) {
+  return item.sellers && item.sellers.filter(checkSKUAvailability).length > 0
+}
+
+function shouldAddToCart(items) {
+  return items.filter(checkAvailableSKUsInSellers).length === 1
```

Comment:
I think the logic here is even simpler.

Just `items.length > 1`

Reply:
But if it is bigger than 1 you have more than one SKU, so you can't add a random SKU to the cart

Reply:
does it add to cart if the item has, say, 3 SKUs, but only 1 available? if so, I don't think this behaviour is desired.

Reply:
It does, the function `checkSKUAvailability` does this check.

Reply:
(talking about this behavior over DM)